src/views/Home/Home.test.jsx

```diff
@@ -0,0 +1,141 @@
+    hdurl: 'https://apod.nasa.gov/apod/image/2101/WetCollodionLunar112820SMO.jpg',
+    media_type: 'image',
+    service_version: 'v1',
+    title: '21st Century Wet Collodion Moon',
+    url: 'https://apod.nasa.gov/apod/image/2101/WetCollodionLunar112820SMO_1024.jpg',
+  },
+  {
+    copyright: 'Hallgrimur P. Helgason Rollover Annotation: Judy Schmidt',
+    date: '2021-01-03',
+    explanation:
+      "All of the other aurora watchers had gone home. By 3:30 am in Iceland, on a quiet September night, much of that night's auroras had died down. Suddenly, unexpectedly, a new burst of particles streamed down from space, lighting up the Earth's atmosphere once again. This time, surprisingly, pareidoliacally, the night lit up with an  amazing shape reminiscent of a giant phoenix. With camera equipment at the ready, two quick sky images were taken, followed immediately by a third of the land.  The mountain in the background is Helgafell, while the small foreground river is called Kaldá, both located about 30 kilometers north of Iceland's capital Reykjavík. Seasoned skywatchers will note that just above the mountain, toward the left, is the constellation of Orion, while the Pleiades star cluster is also visible just above the frame center.  The 2016 aurora, which lasted only a minute and was soon gone forever --  would possibly be dismissed as an fanciful fable -- were it not captured in the featured, digitally-composed, image mosaic.    Almost Hyperspace: Random APOD Generator",
+    hdurl: 'https://apod.nasa.gov/apod/image/2101/PhoenixAurora_Helgason_3130.jpg',
+    media_type: 'image',
+    service_version: 'v1',
+    title: 'A Phoenix Aurora over Iceland',
+    url: 'https://apod.nasa.gov/apod/image/2101/PhoenixAurora_Helgason_960.jpg',
+  },
+];
+test('making sure the correct Api response comes back when using the search feature', async () => {
+  render(<App />);
+  server.use(
+    rest.get('https://api.nasa.gov/planetary/apod', (req, res, ctx) => {
+      const params = req.url.searchParams.get('start_date');
+      // console.log(req.url.toString());
+      if (params) {
+        return res(ctx.json(searchData));
+      } else {
+        return res(ctx.json(data));
```

Comment:
I might send an error instead of this data in the case of query params not being present.